packages/isomorphic/src/internal/normInput.ts

```diff
@@ -0,0 +1,5 @@
+import { Input } from './index'
+
+export default function normInput(input: Input) {
```

Comment:
I think something needs to be adjusted in the TS settings here, because it should be erroring on the fact that this doesn't have a return type.

Reply:
`ripple-binary-codec`,`ripple-keypairs`, and `@xrplf/isomorphic` have `explicit-function-return-type` turned off.  I will turn it on for `@xrplf/isomorphic`.

Reply:
IIRC the typing in `ripple-binary-codec` and `ripple-keypairs` isn't great.